<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SETwistHelixEditorGUIClass</class>
 <widget class="QWidget" name="SETwistHelixEditorGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>188</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>188</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSpinBox" name="spnTurns">
       <property name="minimum">
        <number>-99</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="rdoMinus">
       <property name="text">
        <string> -1BP</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="rdoTwistTurns">
       <property name="text">
        <string>Twist Turns</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="rdoTwistAngle">
       <property name="text">
        <string>Twist Angle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rdoPlus">
       <property name="text">
        <string>+ 1BP</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="spnAngle">
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spnAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onTwistAngleChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnTurns</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onTwistTurnsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoPlus</sender>
   <signal>toggled(bool)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onPlus(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoMinus</sender>
   <signal>toggled(bool)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onMinus(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoTwistAngle</sender>
   <signal>toggled(bool)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onTwistAngle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoTwistTurns</sender>
   <signal>toggled(bool)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onTwistTurn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onMinus(bool)</slot>
  <slot>onPlus(bool)</slot>
  <slot>onTwistAngle(bool)</slot>
  <slot>onTwistTurn(bool)</slot>
  <slot>onTwistAngleChanged(double)</slot>
  <slot>onTwistTurnsChanged(int)</slot>
 </slots>
</ui>
